examples/CalendarCardExpansionExample.m

```diff
@@ -48,9 +48,7 @@ - (void)didTap {
   animator.shouldReverseValues = !_expanded;
   animator.beginFromCurrentState = YES;
 
-  [animator animateWithTraits:traits.navigationBarY animations:^{
```

Comment:
Please remove `CalendarChipTiming.navigationBarY` if it's no longer used (it appears so).

Reply:
Done.